packages/swingset-theme-hashicorp/src/components/live-component/get-file-map.ts

```diff
@@ -0,0 +1,36 @@
+import type { SandpackFile } from '@codesandbox/sandpack-react'
+import { parseCode } from '../code-block/helpers'
+
+export const getFileMap = (codeSnippets: React.ReactElement[]) => {
```

Comment:
```suggestion
export const getFileMap = (codeSnippets: React.ReactNode[]) => {
```

I _think_ this would be the correct type.